### PR TITLE
Tidy up tests more tests

### PR DESCRIPTION
**Pull request by dneise**
This is branched off of @calispac s #34.

I just tidied up the tests file a bit and added a test for the event ids.

This is nothing special, so I would like to release it together with #34 as v1.0.1

### CODE REVIEW

File: protozfits/tests/test_R1_example_file.py [hunk 1]
```diff
@@ -13,20 +14,16 @@
...
-
 def test_can_iterate_over_events_and_run_header():
-    from protozfits import SimpleFile
-    f = SimpleFile(example_file_path)
 
-    camera_config = next(f.CameraConfig)
-    assert (camera_config.expected_pixels_id == np.arange(14)).all()
+    with SimpleFile(example_file_path) as f:
+
+        camera_config = next(f.CameraConfig)
```
Comment (calispac) [inline protozfits/tests/test_R1_example_file.py]: why do we have access the header with `next()` ?

Comment (dneise) [reply protozfits/tests/test_R1_example_file.py]: https://github.com/cta-sst-1m/protozfitsreader#runheader

Comment (calispac) [reply protozfits/tests/test_R1_example_file.py]: ok then is it worth adding a # TODO in the code ? or a Xfail test if one tries to access the header directly ?

Comment (dneise) [reply protozfits/tests/test_R1_example_file.py]: I do not understand what you mean by "access the header directly"

Comment (calispac) [reply protozfits/tests/test_R1_example_file.py]: Doing : 
```python
camera_config = f.CameraConfig
print(camera_config.expected_pixels_id) 
```

**Comment by dneise** [comment]
we should open an issue for that discussion, it is independent of this PR. 

The question about how to access this single row in the Table called "RunHeader" is a few weeks old.

**Pull request merged**